TfIdf_BOW_TpFp.py

```diff
@@ -0,0 +1,46 @@
+import re
+from TfIdf_BOW_creator import sentence_processor
+
+def check_true_positive_oligo_sentence(sentence):
+    threshold = 2
+    count = 0
+    countt = 0
+    result = " "
+
+    file1 = open("oligo_words.txt", "r")
```

Comment:
Could you put the reading of {oligo_words,non, non_oligo_content} in functions? If so,  check_true_positive_oligo_sentence could receive as parameters both sets ...

Don't forget to add the text files in a commit!

Reply:
Added reading functions